packages/react-native-executorch/common/rnexecutorch/models/ocr/Types.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <array>
+#include <string>
+
+namespace rnexecutorch {
+
+struct Point {
```

Comment:
Consider adding explicit constructor here 
`explicit Point(int x_, int y_) : x(x_), y(y_) {}`
to

Reply:
Not needed, resolving